Models/Database/Wallet.cs

```diff
@@ -1,11 +1,14 @@
-﻿using System;
+﻿using CoviIDApiCore.V1.Attributes;
+using System;
 using System.Collections.Generic;
 
 namespace CoviIDApiCore.Models.Database
 {
     public class Wallet : BaseModel<Guid>
     {
+        [Encrypted]
```

Comment:
This needs to be [Encrypted(true)] if I'm not wrong

Reply:
@andrew-delta

Reply:
Correct. All encryption on Wallet should be `[Encrypted(true)]`, while all encryption on WalletDetail should be `[Encrypted]`.

So we should add the `(true)` to this line and the line below.